pkgs/development/ocaml-modules/crowbar/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildDunePackage, fetchFromGitHub, ocplib-endian, cmdliner, afl-persistent
+, calendar, xmldiff, fpath, pprint, uutf, uunf, uucp }:
+
+buildDunePackage rec {
+  pname = "crowbar";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner  = "stedolan";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0wjfc9irvirfkic32ivvj6qb7r838w08b0d3vmngigbjpjyc9b14";
+  };
+
+  minimumOCamlVersion = "4.08";
+
+  propagatedBuildInputs = [ ocplib-endian cmdliner afl-persistent ];
+  checkInputs = [ calendar xmldiff fpath pprint uutf uunf uucp ];
```

Comment:
I believe that `xmldiff` is not maintained. I seems better to just disable one test (e.g., `rm -rf examples/xmldiff/`) instead of packaging an old version of `xmlm` etc.

Reply:
good suggestion, dropped those packages.